docs/source/federation/core-concepts.md

```diff
@@ -86,31 +127,30 @@ What is nice about this model is that we end up with a schema that represents a
 }
 ```
 
-So to review: to reference an external entity from a resolver, we
-1. Define a stub type for the entity we want to reference. The key as declared in the stub must match the entity's own declaration.
-2. From the referencing resolver, return a representation.
-3. In the referenced service, implement a reference resolver for the entity.
+## Extending
+
+An implementing service can add fields to an entity that's defined in another service. This is called **extending** the entity.
 
-## Extending external types
+When a service extends an entity, the entity's _originating_ service is not aware of the added fields. Only the _extending_ service (along with Apollo Gateway) knows about these fields.
```

Comment:
I'm rather partial to "originating" because it describes what the type or field does, as opposed to what the service does.

This feels important in the case of a field in particular. Because it can be argued that the _type_ owns the field. But that can be true regardless of which service the field _originates_ in.